10_code/threshold.py

```diff
@@ -114,23 +114,28 @@
 Texas, deaths from 2003 to 2014, threshold is 2007
 Control states = "TX","UT","GA","CO", "CA","ND","IL", "LA","MD","OK",
 """
-group_tx = ["TX","UT","GA","CO", "CA","ND","IL", "LA","MD","OK"]
-group_tx_names = ["Texas","Utah","Georgia","Colorado", "California","North Dakota","Illinois","Louisiana","Maryland","Oklahoma"]
+texas_states = ["TX","UT","GA","CO", "CA","ND","IL", "LA","MD","OK"]
+texas_names = ["Texas","Utah","Georgia","Colorado", "California","North Dakota","Illinois","Louisiana","Maryland","Oklahoma"]
 
-deaths_tx = overdose_filtered_no_missing[(overdose_filtered_no_missing["State"].isin(group_tx))]
+
+deaths_tx = overdose_filtered_no_missing[(overdose_filtered_no_missing["State"].isin(texas_states))]
 
 # for Texas:
-#2003 to 2005 population data
+#2003 to 2005 old population data
 pop_counties_old = pd.read_csv(
     "https://www2.census.gov/programs-surveys/popest/datasets/2000-2006/counties/totals/co-est2006-alldata.csv", usecols=["stname","ctyname", "popestimate2003","popestimate2004","popestimate2005"])
 
-pop_old = pop_counties_old[(pop_counties_old["stname"].isin(group_tx_names)) & (~ (pop_counties_old["ctyname"].isin(group_tx_names)))]
+pop_old = pop_counties_old[(pop_counties_old["stname"].isin(texas_names)) & (~ (pop_counties_old["ctyname"].isin(texas_names)))]
 
 pop_tx_old = pd.melt(pop_old, id_vars=["stname","ctyname",], var_name="year", value_name="population")
 
 pop_tx_old["year"] = pop_tx_old.year.str.replace("popestimate","")
 pop_tx_old["year"] = pop_tx_old.year.str.strip()
-pop_tx_old["year"].value_counts() #884 counties ✅
+pop_tx_old["year"].value_counts() 
```

Comment:
do you want to assert that we have only 3 years (2003 to 2005) not necessarily doing value counts?